scripts/merge-and-build.sh

```diff
@@ -124,25 +124,19 @@ if [ "${OSE_VERSION}" == "${OSE_MASTER}" ] ; then
   last_tag="$( git describe --abbrev=0 --tags )"
 
   sanity_check
-  # Reset the last three commits and pick up only the tito diff.
-  git reset HEAD~3
-  git add .tito/ origin.spec
-  git commit -m "[CARRY][BUILD] Specfile updates"
-  # Drop the previous web console diff - will be regenerated below.
-  git checkout -- pkg/assets/bindata.go pkg/assets/java/bindata.go
-  set +e
-  # Do not error out for now because these tags already exist due to master (we are testing on fake-master)
-  git tag "${last_tag}" HEAD
-  set -e
 
   echo
   echo "=========="
   echo "Merge origin into ose stuff"
   echo "=========="
 ## Switch back once master is 3.6
 #  git merge -m "Merge remote-tracking branch upstream/master" upstream/master
-  git rebase upstream/release-1.5
+  GIT_SEQUENCE_EDITOR=rebase.py git rebase -i upstream/release-1.5
```

Comment:
Doesn't seem like it's needed. I am testing it locally and it works fine as is. If you feel it's cleaner to separate the two steps I can do it.